### PR TITLE
getdns 1.0.0

### PR DESCRIPTION
**Pull request by joelpurra**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by joelpurra** [comment]
See [getdns-1.0.0 release accouncement](https://getdnsapi.net/releases/getdns-1-0-0/)

Note: might also want to add the [getdns-1.0.0.tar.gz.asc](https://getdnsapi.net/releases/getdns-1-0-0/getdns-1.0.0.tar.gz.asc) PGP signature signed by willem@nlnetlabs.nl, key id [E5F8F8212F77A498](http://pgp2.cs.uu.nl/stats/e5f8f8212f77a498.html).

**Comment by joelpurra** [comment]
As the pull request hasn't been accepted -- possibly because the build checks seem to fail -- I'll use this pull request for a few experiments. Hope you don't mind.

**Comment by joelpurra** [comment]
Also see more info in https://github.com/getdnsapi/getdns/issues/209 and the original request in https://github.com/getdnsapi/getdns-node/issues/8.

**Comment by fxcoudert** [comment]
Devel build fails with `libtool:   error: 'locks.lo' is not a valid libtool object`, after having compiled `locks.c` several times in parallel.

**Comment by joelpurra** [comment]
@fxcoudert: thanks for looking at it! I'll alert the getdns developers in https://github.com/getdnsapi/getdns/issues/209.

Just pushed audit/style fixes, in case you wondered.

**Comment by fxcoudert** [comment]
For a non-parallel build, please add a line with `ENV.deparallelize` before calling `make`

**Comment by joelpurra** [comment]
@fxcoudert: thanks!

@wtoorop: here you go =)

File: Formula/getdns.rb [hunk 1]
```diff
@@ -14,14 +13,30 @@ class Getdns < Formula
...
+  if build.head?
+    depends_on "libtool"
+    depends_on "autoconf"
+    depends_on "automake"
+  end
+
   def install
+    if build.head?
+      system "glibtoolize", "-ci"
+      system "autoreconf", "-fi"
```
Comment (joelpurra) [inline Formula/getdns.rb]: @wtoorop: assuming brew takes care of it with this log output:

```text
==> Checking out branch develop
git checkout -f develop --
Already on 'develop'
Your branch is up-to-date with 'origin/develop'.
git reset --hard origin/develop
HEAD is now at fe49bc1 Merge pull request #279 from dkg/feature/padding-policy
git submodule foreach --recursive git submodule sync
Entering 'src/jsmn'
git submodule update --init --recursive
```

**Comment by dunn** [comment]
@BrewTestBot test this please

**Comment by joelpurra** [comment]
@dunn: any idea why only the `--only-dependencies` tests are run? There should be more, right?

**Comment by dunn** [comment]
I guess it's stopping there because `postinstall` fails for one of the deps:
```
/usr/bin/sandbox-exec -f /tmp/homebrew20170401-12823-1cbf918.sb nice /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby -W0 -I /usr/local/Homebrew/Library/Homebrew -- /usr/local/Homebrew/Library/Homebrew/postinstall.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/unbound.rb --only-dependencies --build-bottle --verbose
Warning: The post-install step did not complete successfully
You can try again using `brew postinstall unbound`
```

**Comment by joelpurra** [comment]
@dunn: so the issue not directly related to `getdns.rb` but [`unbound.rb`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/unbound.rb)?

- The last change to `unbound.rb` is over two months old. See most recent [`unbound.rb` commit from 2017-02-01](https://github.com/Homebrew/homebrew-core/blob/bab33c0e4a7f439abec9c76ecd1116ac1f1ffd17/Formula/unbound.rb).
- The unbound post-install warning (error) doesn't look intermittent.
- Should I open a new issue?

**Comment by dunn** [comment]
> Should I open a new issue?

I guess so, thanks.  I'm surprised it didn't show up until now; I guess not a lot of things depend on unbound.

**Comment by joelpurra** [comment]
@dunn: done!

See #11998 [Post-install problems for unbound as a dependency](https://github.com/Homebrew/homebrew-core/issues/11998).

**Comment by joelpurra** [comment]
After @MikeMcQuaid fixed `unbound` in #11998/#12093 (thanks!) the build passes. Merge?

**Comment by MikeMcQuaid** [comment]
Thanks again for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by joelpurra** [comment]
@MikeMcQuaid: wow, thanks! You're the one deserving the praise though =)

Thanks for the merge! This also closes https://github.com/getdnsapi/getdns/issues/209 and https://github.com/getdnsapi/getdns-node/issues/8. Cheers =)

**Pull request closed without merging**